utils/backend/mysk/useMySKClient.ts

```diff
@@ -26,7 +26,8 @@ export default function useMySKClient(): MySKClient {
 
   useEffect(() => {
     // Fetch the user data if not yet fetched.
-    if (loading || user || person) return;
+    if (!document.cookie.includes("access_token") || loading || user || person)
```

Comment:
`!document.cookie.includes("access_token")`: Good catch if the user has no `access_token`!